client/my-sites/checkout/checkout/subscription-text.jsx

```diff
@@ -15,14 +16,27 @@ import { cartItems } from 'lib/cart-values';
 
 class SubscriptionText extends React.Component {
 	render() {
-		if ( cartItems.hasRenewalItem( this.props.cart ) ) {
-			return (
-				<span className="subscription-text">
-					{ this.props.translate( 'renews annually', {
-						context: 'Informative text for renewals in /checkout',
-					} ) }
-				</span>
-			);
+		const { cart, translate } = this.props;
+
+		if ( cartItems.hasRenewalItem( cart ) ) {
+			const product = cart.products[ 0 ];
+			let informative_text = '';
+
+			if ( isBiennially( product ) ) {
+				informative_text = translate( 'renews biennially', {
```

Comment:
IMHO many people get confused by this term, not knowing positively whether it means once every two years or twice per year. If we have the space I would recommend changing this to "renews every two years".

Reply:
Done :-).